examples/cifar10/pt/learners/cifar10_learner.py

```diff
@@ -155,17 +156,19 @@ def _initialize_trainer(self, fl_ctx: FLContext):
 
         self.valid_loader = torch.utils.data.DataLoader(self.valid_dataset, batch_size=64, shuffle=False, num_workers=2)
 
+        self.initialized = True
+
     def _terminate_trainer(self):
         # collect threads, close files here
         pass
 
     def handle_event(self, event_type: str, fl_ctx: FLContext):
         # the start and end of a run - only happen once
-        if event_type == EventType.START_RUN:
+        if event_type == EventType.START_RUN:  # TODO: Is this still needed? Clara's FederatedLearner just used init in train/val functions
```

Comment:
You shouldn't handle START_RUN event. Instead you should implement the initialize() method of the Learner. Two reasons:
- This event is handled by the executor and it calls the initialize() method of the Learner;
- The initialize() method provides additional "parts" to you, which could be useful. For example, it could be configured to give you the components to stream error and/or TB event data to server.

Reply:
Ok. I implemented the initialize() method now.